configs/openmrs_config/ampathforms/NCD_Consultation.json

```diff
@@ -32,7 +32,33 @@
 							},
 							"type": "obs",
 							"validators": []
-						}
+						},
+						{
+                            "id": "TB symptoms",
```

Comment:
@kazlaw this should be one word like `TBSymptoms`

Reply:
Thanks @reagan, let me try it out